README.md

```diff
@@ -95,3 +95,9 @@ target/release/drml \
 	--alice \
 	--base-path tests/local-testnet/alice
 ```
+
+#### Build in Nixos
```

Comment:
This part is under the `### Pangoro Testnet`

Reply:
Remove or put it in somewhere like `## Install ### nix-pkg`

Reply:
ACK